src/main/java/com/fismobile/appium/pages/AboutPage.java

```diff
@@ -2,42 +2,35 @@
 
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.PageFactory;
-import io.appium.java_client.pagefactory.AppiumFieldDecorator;
 import org.openqa.selenium.support.FindBy;
-import com.fismobile.appium.utility.GlobalParameters;
 /*
  *
  * @author wloforte
  */
-public class AboutPage {
+public class AboutPage extends BasePage {
 
     @FindBy(id = "i_am_an_id")
     private WebElement title;
 
     @FindBy(id = "i am a link")
-    private WebElement span;
+    private WebElement link;
 
     @FindBy(id = "comments")
     private WebElement txt_comments;
 
-    public AboutPage(WebDriver driver) {
-
-        if(GlobalParameters.runType.equalsIgnoreCase("web"))
-        {
-            PageFactory.initElements(driver, this);
-        }else
-        {
-            PageFactory.initElements(new AppiumFieldDecorator(driver), this);
-        }
-
-    }
+    public AboutPage(WebDriver driver) { super(driver); }
 
     public String getTitle() {
         return title.getText();
     }
 
-    public String getSpan() { return span.getText(); }
+    public void clickLink() { link.click(); }
+
+    public AnotherPage goToAnother() throws InterruptedException {
```

Comment:
is this name correct? Is it "going" to AnotherPage or "creating" it?

Reply:
Both. From the test point of view is navigating to the other page, the function clicks in an element and returns an instance of the page.